isyfact-standards-doc/src/docs/antora/modules/isy-bedienkonzept/pages/bedienkonzept.adoc

```diff
@@ -534,6 +538,7 @@ Aktive Elemente sind in der Highlightfarbe und hinterlegt dargestellt.
 *	Sollten Listen viele Einträge haben, sorge mit anderen Elementen für die Durchsuchbarkeit, Filterung oder Kategorisierung.
 *	Listen helfen bei komplexen Eingaben Fehler zu vermeiden.
 *	Kombiniere Listen mit unterschiedlichen interaktiven Elementen, um dem Nutzer eine Hilfestellung zu bieten.
+* Erwäge die Nutzung einer Radio-Button-Gruppe bei einer kleinen Liste.
```

Comment:
Nur ein Leerzeichen bei Aufzählungen (bei allein Do's and Don'ts). Hier ein Beispiel::
```suggestion
* Sollten Listen viele Einträge haben, sorge mit anderen Elementen für die Durchsuchbarkeit, Filterung oder Kategorisierung.
* Listen helfen bei komplexen Eingaben Fehler zu vermeiden.
* Kombiniere Listen mit unterschiedlichen interaktiven Elementen, um dem Nutzer eine Hilfestellung zu bieten.
* Erwäge die Nutzung einer Radio-Button-Gruppe bei einer kleinen Liste.
```